analysis/README.md

```diff
@@ -4,6 +4,45 @@ This folder contains implementation of Edna data analysis in Python.
 
 ## Usage
 
-First of all, normally you don't want to call this code directly, it is supposed to be called from `edna-server`.
-We use [`poetry`](https://python-poetry.org/) for dependency management, though you can install dependencies using your favorite way and not use `poetry`.
+First, normally you don't want to call this code directly, it is supposed to be called from `edna-server`.
+We use [`poetry`](https://python-poetry.org/) for dependency management, though you can install dependencies using your
+favorite way and not use `poetry`.
 To get shell with all dependencies installed run `poetry install` and then `poetry shell` from this directory.
+
+## Features
+
+The list of features implemented in an analytic module:
+1. Compute parameters of fitted sigmoid.
+2. Detect potential outliers.
+3. Compute parameters of fitted sigmoid without outliers.
+
+### Input format
+
+JSON containing list of experiments in the form:
+```
+[{"experiment": int,
+"data": [[float, float], ...],
+"findOutliers": boolean},
+...]
+```
+
+### Output format
+
+JSON containing list of calculated values per each experiment:
+```
+[{"experiment": int,
+"params": [float, float, float, float],
+"outliers": [int, ...],
+"new_params": [float, float, float, float],
+"status": str},
+...]
+```
+`experiment` and `status` are always present in the output. 
```

Comment:
```suggestion
`experiment` and `status` are always present in the output.
```

Reply:
why does git not show this particular whitespace beforehand?( Also what's the tool to use to detect them automatically?

Reply:
https://github.com/serokell/serokell.nix/blob/master/overlay/default.nix#L30

Reply:
> why does git not show this particular whitespace beforehand?

:shrug: 

> Also what's the tool to use to detect them automatically?

Nearly all IDEs/editors have a builtin option or a plugin to remove trailing whitespaces on save. If you google "remove trailing whitespaces <your editor/IDE>", most likely you'll find a solution.

Reply:
It was ON, but didn't delete for .md file for some reason(